examples/apache_libcloud/public-cloud-modular_instance.py

```diff
@@ -0,0 +1,221 @@
+"""Script used to import not monitored servers from cloud into Cyberwatch and delete terminated cloud servers"""
+
+# Based on the Apache Libcloud API https://libcloud.apache.org/index.html
+# Tested on Google Compute Engine and AWS EC2
+
+# Prerequisites :
+# - Install libcloud with command "pip3 install apache-libcloud"
+# - Choose a driver (EC2/GCE) and set the corresponding variables (https://libcloud.readthedocs.io/en/stable/compute/drivers/)
```

Comment:
Split into two comments. The second explains that there are constants to set in the first lines of the script depending on the cloud provider.